recipe/meta.yaml

```
@@ -23,14 +23,14 @@ source:
     sha256: 61abdcdb13684ba2a3ba4afb7ea6c7907aa0896a46439d3af7e8848483d4392f
     folder: dolfin
     patches:
-      - boost.patch
+      - libboost-python-devel.patch
```

Ah, that patch name tripped up the automation 😅 
```suggestion
      - boost.patch
```